rc_vars.yaml

```diff
@@ -28,8 +28,9 @@ coredns_version: "v1.7.0"
 helm_version: "v3.5.4"
 kube_apiserver_port: 443
 enable_nodelocaldns: false
-weave_version: 2.8.1
-weave_kube_image_repo: "{{ docker_image_repo }}/weaveworks/weave-kube"
-weave_kube_image_tag: "{{ weave_version }}"
+#### for now, we're going with the default version defined under roles/download/defaults/main.yml (2.7.0)
+# weave_version: 2.8.1
+# weave_kube_image_repo: "{{ docker_image_repo }}/weaveworks/weave-kube"
```

Comment:
dont you have to supply our own weave repo?

Reply:
rc_vars.yaml
  docker_image_repo: "certified-registry.rcplatform.io/public"

roles/download/defaults/main.yml
  weave_kube_image_repo: "{{ docker_image_repo }}/weaveworks/weave-kube"
  weave_kube_image_tag: "{{ weave_version }}"
  weave_npc_image_repo: "{{ docker_image_repo }}/weaveworks/weave-npc"
  weave_npc_image_tag: "{{ weave_version }}"